coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/log/RequestResponseLogger.java

```diff
@@ -377,17 +384,24 @@ protected String printResponseEntity(Object entity, Integer maxLogSize) {
      *            log prefix
      * @param maskingNeeded
      *            is masking sensitive data needed
+     * @param mediaType
+     *            media type of entity if relevant
+     * @return entity in string
      */
-    public String printEntity(Object entity, Integer maxLogSize, String prefix, boolean maskingNeeded) {
+    public String printEntity(Object entity, Integer maxLogSize, String prefix, boolean maskingNeeded, MediaType mediaType) {
         if (entity == null) {
             return null;
         }
         StringBuffer sb = new StringBuffer();
         String entityText = null;
         if (entity instanceof String) {
             entityText = (String) entity;
-        } else {
+        } else if (mediaType != null && StringUtils.containsIgnoreCase(mediaType.getSubtype(), "json")) {
             entityText = JsonUtil.toJson(entity);
+        } else if (mediaType != null && StringUtils.containsIgnoreCase(mediaType.getSubtype(), "xml")) {
+            entityText = MarshallingUtil.marshall(entity);
```

Comment:
Ez remelem nem JAXB_FORMATTED_OUTPUT, mert jo lenne ha egy sorban printelne, ne szemetelje es torzitja el a logot. Mindenki a logokban a hibaknal a stacktracet nezi, es jo lenne ez nem keverne vele.

Reply:
Megnéztem, és ez szépen egy sorba írja ki:
Példa: entity: 
```
[<?xml version="1.0" encoding="UTF-8" standalone="yes"?><BaseResponse xmlns="http://common.dto.coffee.icellmobilsoft.hu/commonservice"><context><requestId>TEST</requestId><timestamp>2020-09-14T12:42:19.599956+02:00</timestamp></context><funcCode>OK</funcCode><message>msg</message></BaseResponse>]
```
Gyorsan erre is írok egy tesztet, hogy látható legyen, biztosan.

Reply:
+ migracio doksi

Reply:
Még nem volt 1.1.0->1.2.0 migrációs doksi, létrehoztam. Viszont ezt a feladatot megvárnám hogy bemenjen és akkor kiegészítem a másik feladatban a migrációs doksit, hogy ne legyen conflict,